data/test/scenarios/wml_tests/UnitsWML/test_movetype.cfg

```diff
@@ -442,36 +442,36 @@
 )}
 
 #####
-# API(s) being tested: [store_locations][filter_vision]
+# API(s) being tested: [movetype]
```

Comment:
For this one (the first of the `MOVETYPE_VISION_SCEN` ones), I'd suggest `[movetype],[store_locations][filter_vision]`. This one is partly here to sanity-check the framework that it's using, and it's definitely checking `[store_locations][filter_vision]`'s main function - that the API works with fog.

Reply:
I'd call this one (similar to my other replies) as relying on `[store_locations][filter_vision]`, but primarily testing `[movetype]`. Or since this is just meant as a baseline test without having changed the movetype at all, it could just be N/A.

Reply:
Changed to N/A.